### ISSUE TITLE
archlinux build error

### ISSUE THREAD

**Issue by smldis** [rdeits/DrakeVisualizer.jl#49]
lsb_release command outputs "rolling" and VersionNumber cannot parse it.
The error is:
```
============================================================================================================================================
INFO: Building DrakeVisualizer
=========================================================[ ERROR: DrakeVisualizer ]=========================================================

LoadError: ArgumentError: invalid version string: rolling
while loading /home/smldis/.julia/v0.6/DrakeVisualizer/deps/build.jl, in expression starting on line 21

============================================================================================================================================
```

**Comment by rdeits** [comment]
Thanks for reporting this. I think it should be fixed by #51

**Comment by rdeits** [comment]
@smldis I don't have any non-ubuntu systems, so can you please test if this is fixed in master?
